contrib/patterns/_grid-media.scss

```diff
@@ -0,0 +1,16 @@
+$medium-screen: 1000px;
+$medium: "only screen and (min-width:#{$medium-screen})";
```

Comment:
@tysongach maybe we could do something like a parse media query. It'd be a pain in the but to write so i want to check first but maybe something like the following:

1. `grid-media` passes the first parameter to some sort of `_parse-media` function goes and says

```scss
$neat-grid: (
  columns: 12,
  gutter: 20px,
  media: min-width, // defaults to `min-width` but could be set to `max-width`
);

@function _parse-media($var)
  if type($var) == number {
    @return "(only screen and #{$neat-grid:media}: {$var} )";
  } @else if type($var) == string {
    @return "(only screen and #{$neat-grid:media}: {$var} )";
  }
}
```

and then in turn `grid-media` does something like:

```scss
@mixin grid-media($query, $grid: $neat-grid) {
  @media #{ _parse-media($query)} {
    …
  }
}
```

the idea being that you can

1. you can set the default to be mobile first or desktop first
2. because of this default you are able to just pass in a number for basic media queries like `@include media(960px) { … }` and you are good to go.

there is no global mq settings, a la `new-break-point` but i'm cool with that. This is an area which could get complex very quickly but I think solves the issue *relatively* well. We can always add more features later.